telethon/client/updates.py

```diff
@@ -405,7 +405,10 @@ async def _dispatch_update(self: 'TelegramClient', update, others, channel_id, p
             #
             # It will return `None` if we haven't logged in yet which is
             # fine, we will just retry next time anyway.
-            await self.get_me(input_peer=True)
+            try:
+                await self.get_me(input_peer=True)
+            except (OSError, errors.RPCError):
```

Comment:
What RPCError does this raise? I thought only unauthorized errors could occur, which is what this does and then it would return `None`:

https://github.com/LonamiWebs/Telethon/blob/61b0f09e1ddea5188a3905336698e8b42d42620f/telethon/client/users.py#L164-L165

The `OSError` makes sense for handling lack of connection, but I would like to know what RPC error can occur and if none remove that part.